analysis_options.yaml

```diff
@@ -0,0 +1,8 @@
+analyzer:
+  language:
+    enableStrictCallChecks: true
+    enableSuperMixins: true
+    enableAssertInitializer: true
+  strong-mode:
+    implicit-dynamic: false
+    
```

Comment:
we shouldn't need this file, `flutter analyze` should automatically get the options from the flutter repo.